lib/Kconfig

```diff
@@ -6,22 +6,24 @@
 
 menu "Libraries"
 
-rsource "bin/Kconfig"
+rsource "at_cmd_parser/Kconfig"
```

Comment:
Could you split these changes and those to `lib/CMakeLists.txt` into a separate commit?

Reply:
I think it is easier to have this list sorted. I will to this in a different commit.